packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -874,10 +949,11 @@ export class StatsAnalyzer extends EventsScope {
   private processOutboundRTPResult(result: any, mediaType: any) {
     const sendrecvType = STATS.SEND_DIRECTION;
 
+    this.processTrackResult(result, mediaType, sendrecvType);
```

Comment:
processTrackResult is now called to process inbound or outbound rtp result and not track result, so it should be renamed

Reply:
changed to 'ProcessResolutionsStats'.